### ISSUE TITLE
config.plugins.terminal.arguments does not work on Windows

### ISSUE THREAD

**Issue by takase1121** [adamharrison/lite-xl-terminal#56]
This is apparently the case, but not documented.

Otherwise, the Lua part should at least concatenate the string, or steal Lite XL's code to properly serialize this. My preference is just to document this in plugin settings.

**Comment by adamharrison** [comment]
Yeah just documented for now. I'll fix it on windows next time I boot into there.
